docs/testing.md

```diff
@@ -0,0 +1,98 @@
+# Testing Sematic Pipelines
+
+As a tool for creating
+[Continuous Learning](https://medium.com/sematic-blog/continuous-learning-for-safer-and-better-ml-models-42e51851b4bd)
+pipelines, Sematic fully appreciates the value of tests. Regardless of whether
+your pipeline is handling updates to a production machine learning model,
+manipulating some internal data, or doing regression testing itself, it's
+important to make sure the pipeline itself is healthy. You don't want to
+have your pipeline execute for hours or days, and then have it fail near
+the end for a simple error!
+
+Sematic will perform some up-front checks, like performing some early type
+checking between the inputs and outputs of the funcs in your pipeline.
+However, there's nothing that replaces the value of a good, fast unit test.
+To aid in the creation of pipeline unit tests, Sematic provides
+`sematic.testing.mock_sematic_funcs`. It can be used as follows:
+
+```python
+import pytest
+import sematic
+from sematic.resolvers.silent_resolver import SilentResolver
```

Comment:
Import from the public API entry point:
```python
from sematic import SilentResolver
```

Reply:
`SilentResolver` actually isn't present in the public API entrypoint. Should it be?